addons/l10n_ae/models/account_chart_template.py

```diff
@@ -0,0 +1,47 @@
+    _inherit = 'account.chart.template'
+
+    def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
+        """ If UAE chart, we add 2 new journals TA and IFRS"""
+        if company.country_id.code == "AE":
+            if not journals_dict:
+                journals_dict = []
+            journals_dict.extend(
+                [{"name": "Tax Adjustments", "company_id": company.id, "code": "TA", "type": "general", "sequence": 1,
+                  "favorite": True},
+                 {"name": "IFRS 16", "company_id": company.id, "code": "IFRS", "type": "general", "favorite": True,
+                  "sequence": 10}])
+        return super()._prepare_all_journals(acc_template_ref, company, journals_dict=journals_dict)
+
+    def _load_template(self, company, code_digits=None, account_ref=None, taxes_ref=None):
+        account_ref, taxes_ref = super(AccountChartTemplate, self)._load_template(company=company,
+                                                                                  code_digits=code_digits,
+                                                                                  account_ref=account_ref,
+                                                                                  taxes_ref=taxes_ref)
+        if company.country_id.code == "AE" and self.get_xml_id().get(1) == "l10n_ae.uae_chart_template_standard":
+            ifrs_journal = self.env['account.journal'].search(
+                [('company_id', '=', company.id), ('code', '=', 'IFRS')]).id
+            ifrs_accounts = self.env['account.account'].search([('company_id', '=', company.id),
+                                                                ('code', 'in',
+                                                                 (self.env.ref("l10n_ae.uae_account_100101").code,
+                                                                  self.env.ref("l10n_ae.uae_account_100102").code,
+                                                                  self.env.ref("l10n_ae.uae_account_400070").code))])
+            for account in ifrs_accounts:
+                account.allowed_journal_ids = [(4, ifrs_journal, 0)]
+
+            tax_payable_account_id = self.env['account.account'].search([('company_id', '=', company.id),
+                                                                         ('code', '=', self.env.ref(
+                                                                             "l10n_ae.uae_account_202003").code)])
+            tax_receivable_account_id = self.env['account.account'].search([('company_id', '=', company.id),
+                                                                            ('code', '=', self.env.ref(
+                                                                                "l10n_ae.uae_account_100103").code)])
+            if tax_payable_account_id:
+                self.env.ref('l10n_ae.ae_tax_group_5').property_tax_payable_account_id = tax_payable_account_id.id
```

Comment:
It is the only tax group with closing accounts

Reply:
Probably because it is the only one with amounts?  But then what is the difference with putting a generic one on the chart of accounts?

Reply:
The other two groups are `VAT 0%` and `VAT Exempt` so if we add accounts to the other two we will have entries with 0 VAT show up on the VAT payable and receivable accounts.

Reply:
@sal-odoo And that is a big problem?  (no problem to do it this way, but let's make sense to do so)

Reply:
@jco-odoo according to my consultant, because the other groups include a reverse charge tax which should be closed separately we should not add the same closing accounts to the other groups